### ISSUE TITLE
Promoted cards block is overlapping system messages

### ISSUE THREAD

**Issue by alanmels** [backdrop/backdrop-issues#6220]
## Description of the bug

For some reason, the promoted cards block has negative margin on top:

```
.l-top .block-views-promoted-cards-block {
  margin-top: -2rem;
} 
```

and it makes it overlap the messages block like so:

![Screenshot 2023-09-14 at 4 21 48 PM](https://github.com/backdrop/backdrop-issues/assets/56604405/0ef66d01-1a1c-4e2b-a7ab-52971c6aee9a)

## Steps To Reproduce

To reproduce the behavior:

1. Just login to freshly installed Backdrop website to see two divisions are overlapping each other.
2. If you navigated away, just flush all caches and you will see the problem again.

![Screenshot 2023-09-14 at 4 35 40 PM](https://github.com/backdrop/backdrop-issues/assets/56604405/9a188bb1-e47f-40b7-8286-05e7a07347d1)

## Expected behavior

Not a big deal, but still aesthetically would be nice if the default theme comes without this kind of minor issues. If the above css rule is fixed then it will look like this:

![Screenshot 2023-09-14 at 4 22 10 PM](https://github.com/backdrop/backdrop-issues/assets/56604405/92c4143c-7e7b-4c15-8880-3d74cec4e7c7)

## Additional information

I would submit a simple PR removing the offending css rule, however for some reason some other divisions on the front page are pulled up for exactly the same amount:

```
.l-top .block-hero {
  margin-top: -2rem;
}
```

Don't want to cause regression, but note that while other divisions look normally, only the promoted cards block division is overlapping.

**Comment by laryn** [comment]
Is this a duplicate of https://github.com/backdrop/backdrop-issues/issues/5769?

**Comment by olafgrabienski** [comment]
Yes, it is. Let's close this one (like we already closed #6074) in favor of #5769, and hopefully fix the issue there soon.
